### PR TITLE
New package: btrfs-backup-0.3.1

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/btrfs-backup/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'btrfs-backup'
+pkgname=btrfs-backup
+_pkgname=btrfs_backup
```
Comment (cr6git) [inline srcpkgs/btrfs-backup/template]: Instead of a custom variable, just could just use: `${pkgname/-/_}`

File: srcpkgs/btrfs-backup/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+build_style=python3-module
+pycompile_module="btrfs_backup"
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools"
+short_desc="Incremental atomic backups for btrfs using snapshots"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="MIT"
+homepage="https://github.com/efficiosoft/btrfs-backup"
+distfiles="${PYPI_SITE}/b/${pkgname}/${_pkgname}-${version}.tar.gz"
+wrksrc="${_pkgname}-${version}"
```
Comment (cr6git) [inline srcpkgs/btrfs-backup/template]: move `wrksrc=` right below `revision=`

File: srcpkgs/btrfs-backup/template (hunk 1 above)
Comment (shizonic) [reply srcpkgs/btrfs-backup/template]: Perfect. Didn't know this. Thanks!

File: srcpkgs/btrfs-backup/template (hunk 2 above)
Comment (shizonic) [reply srcpkgs/btrfs-backup/template]: Alright. Will do this in future.

File: srcpkgs/btrfs-backup/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'btrfs-backup'
+pkgname=btrfs-backup
+version=0.3.1
+revision=1
+wrksrc="${pkgname/-/_}-${version}"
+noarch=yes
+build_style=python3-module
+pycompile_module="${pkgname/-/_}"
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools"
```
Comment (cr6git) [inline srcpkgs/btrfs-backup/template]: https://github.com/efficiosoft/btrfs-backup#requirements

`btrfs-progs` is required, obviously... but it's already part of `base-system` so I don't think we have to add it explicitly.

**Pull request merged**